std_msgs/Header header
SeyondPacket[] packets               # packet contents
